pytest/empty-contract-rs/src/lib.rs

```diff
@@ -1,5 +1,5 @@
-use borsh::{BorshDeserialize, BorshSerialize};
 use near_sdk::{env, metadata, near_bindgen};
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
```

Comment:
why is `self` needed?

Reply:
It's used in some macros (e.g. `derive`).